cmd/pint/lint.go

```diff
@@ -114,6 +119,20 @@ func actionLint(c *cli.Context) error {
 	return nil
 }
 
+func report(summary reporter.Summary, reporters *config.Reporters) error {
+	if reporters != nil {
+		if reporters.JSON != nil {
+			r := reporter.NewJSONReporter(reporters.JSON.Path)
+			err := r.Submit(summary.Reports())
```

Comment:
```suggestion
			if err := r.Submit(summary.Reports()); err != nil { return err; }
```
You can shorthand this too like this^